lib/src/app/cast.js

```diff
@@ -18,58 +20,73 @@ import '../styles/player.css';
  * @param {PageLevelParameters} [pageLevelParameters]
  */
 export function init(namespace, pageLevelParameters) {
-    var mediaManager = null;
-    var castPlayer = new CastPlayer(namespace, pageLevelParameters);
-
     logger.setLevel(process.env.LOG_LEVEL);
     UIManager.setStatusCast(UIManager.status.LOADING);
 
-    // create a new instance of the Media Manager to interact with the sender app request
-    mediaManager = new cast.receiver.MediaManager(castPlayer);
-
-    // backup of the original onLoad event handler to use it after our custom logic
-    mediaManager.origOnLoad = mediaManager.onLoad;
+    const castPlayer = new CastPlayer(namespace, pageLevelParameters);
+    const mediaManager = new cast.receiver.MediaManager(castPlayer);
 
     /**
-     * onLoad assign a new implementation of the onLoad event that can sets the needed information from the
+     * Exotic class method extending by Google devs
+     * @see https://developers.google.com/cast/docs/custom_receiver (section Media)
+     * onLoad assign a new implementation of the onLoad event that can sets the
+     * needed information from the
      * incoming event
      * @param {cast.receiver.MediaManager.Event} event
      */
-    mediaManager.onLoad = function(event) {
-        logger.info('MediaManager:onLoad', event);
-        if (event && event.data && event.data.autoplay) {
-            Timer.setIdle();
-        }
-        castPlayer.setAsset(event.data.media.customData);
-        if (castPlayer.OOPlayer != null) {
-            castPlayer.notifySenders(castPlayer.OOPlayer.getState());
+    mediaManager.onLoad = (() => {
```

Comment:
Please use the old style of function declarations.

Reply:
That goes for the other functions you changed too.

Reply:
I'm actually for using ES6 style function declarations. Arrow functions also have the benefit of binding this to their context.

Reply:
In the functions delcarations of the code styling agreement is point «There is a big mix of how functions are declared throughout the code base. If you are creating a new file, then use this system. Otherwise adopt what is in the file already.»
Arrow functions has been using here from the beginning of the repo: https://github.com/ooyala/chromecast-sample-receiver/blame/master/lib/src/app/idleTimer.js#L27
So I just followed this way, since it's shorter, more descriptive, easier to be managed by GC (anonymous functions being killed with no investigations who refers to it), free of naming collisions.

Reply:
you guys convinced me 👍